packages/zoe/src/contractFacet/zcfSeat.js

```diff
@@ -124,7 +124,7 @@ export const createSeatManager = (
     seats.forEach(assertStagedAllocation);
 
     // Keep track of seats used so far in this call, to prevent aliasing.
-    const zcfSeatsSoFar = new WeakSet();
+    const zcfSeatsSoFar = new Set();
```

Comment:
Good. (Somehow I thought this had already been fixed. Was there a similar fix elsewhere?)

Reply:
I think ERTP is now using a set for anti-aliasing, whereas some months ago it wasn't